pkgs/applications/radio/gridtracker/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, nwjs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gridtracker";
+  version = "1.22.1226";
+
+  src = fetchFromGitLab {
+    owner = "gridtracker.org";
+    repo = "gridtracker";
+    rev = "v${version}";
+    sha256 = "sha256-/Noc2aqHBjphX6RDqxQBI/OOKZgEnOndn0daBt1edXM=";
+  };
+
+  installPhase = ''
+    make DESTDIR=$out NO_DIST_INSTALL=1 install
+    substituteInPlace $out/usr/bin/gridtracker \
+        --replace "exec nw" "exec ${nwjs}/bin/nw" \
+        --replace "/usr/share/gridtracker" "$out/usr/share/gridtracker"
+    mkdir -p $out/bin
+    ln -s $out/usr/bin/gridtracker $out/bin/gridtracker
+  '';
+
+  meta = with lib; {
+    description = "An amateur radio companion to WSJT-X or JTDX";
+    longDescription = ''
+      GridTracker listens to traffic from WSJT-X/JTDX, displays it on a map,
+      and has a sophisticated alerting and filtering system for finding and
+      working interesting stations. It also will upload QSO records to multiple
+      logging frameworks including Logbook of the World.
+    '';
+    homepage = "http://js8call.com/";
```

Comment:
```suggestion
    homepage = "https://gridtracker.org";
```

Reply:
Oops, no prizes for guessing what I was hacking on before this then :facepalm: